specs/lib/removeUnusedVerticesSpec.js

```diff
@@ -1,592 +1,591 @@
-'use strict';

```

Comment:
The line endings in this file were wrong, which is why changing updated every line in the file.  All I did was remove an extra line break, no need to review.